src/components/molecules/ExpiryModal/ExpiryModal.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import styled from 'styled-components';
-import Modal, { TModalProps } from '../Modal/Modal';
+import Modal, { ModalProps } from '../Modal/Modal';
```

Comment:
This is bit stuttery - not for this PR maybe but would be cleaner if we exported these from index.ts in Modal

Reply:
You mean having `ModalProps` in src/index.ts`?